packages/events/src/once.rs

```diff
@@ -0,0 +1,392 @@
+//! Events that can happen at most once (send/receive consumes the sender/receiver).
+//!
+//! This module provides one-time use events based on oneshot channels. Each event can only
```

Comment:
"Based on oneshot channels" sure sounds like an implementation detail. Do not document internal facts that are not impactful to users actually using the API surface.